CREDITS.md

```diff
@@ -394,3 +394,4 @@
  * `support` by Chocobozzz (CC-BY)
  * `language` by Aaron Jin (CC-BY)
  * `video-language` by Rigel Kent (CC-BY)
+ * `peertube-x` by Solen DP (CC-BY)
```

Comment:
Add it in https://github.com/Chocobozzz/PeerTube/blob/develop/scripts/generate-code-contributors.ts#L38 instead, because I use a script to automatically generate `CREDITS.md`